src/common/SPELLS/deathknight.ts

```diff
@@ -301,6 +301,12 @@ const spells = spellIndexableList({
     icon: 'achievement_boss_svalasorrowgrave',
   },
 
+  SUMMON_GARGOYLE: {
+    id: 49206,
+    name: 'Summon Gargoyle',
+    icon: 'ability_deathknight_summongargoyle',
+  },
+
```

Comment:
there should be a `TALENTS` spell for this. don't need to add a `SPELLS` entry

Reply:
(unless the spell ID in the talent doesn't match what is actually cast---this happens sometimes)

Reply:
I tried out using TALENTS.SUMMON_GARGOYLE_TALENT in the cast event and it didn't work unfortunately.

Reply:
👍

Reply:
@Bicepspump Did you use just the talent or the talent + the arbiter glyph cast?

Reply:
Ah, you're right. Garg talent and Arbiter spell worked!